src/NavBar.php

```diff
@@ -12,174 +9,281 @@
 /**
  * NavBar renders a navbar HTML component.
  *
- * Any content enclosed between the [[begin()]] and [[end()]] calls of NavBar
- * is treated as the content of the navbar. You may use widgets such as [[Nav]]
- * or [[\yii\widgets\Menu]] to build up such content. For example,
+ * Any content enclosed between the {@see begin()} and {@see end()} calls of NavBar is treated as the content of the
+ * navbar. You may use widgets such as {@see Nav} or {@see \Yiisoft\Widget\Menu} to build up such content. For example,
  *
  * ```php
- * use Yiisoft\Yii\Bootstrap4\NavBar;
- * use Yiisoft\Yii\Bootstrap4\Nav;
+ *    if ($user->getId() !== null) {
+ *        $menuItems = [
+ *            [
+ *                'label' => 'About',
+ *                'url' => '/about',
+ *            ],
+ *            [
+ *                'label' => 'Contact',
+ *                'url' => '/contact',
+ *            ],
+ *            [
+ *                'label' => 'Logout' . ' ' . '(' . $user->getUsername() . ')',
+ *                'url' => '/logout'
+ *            ],
+ *        ];
+ *    } else {
+ *        $menuItems = [
+ *            [
+ *                'label' => 'About',
+ *                'url' => '/about',
+ *            ],
+ *            [
+ *                'label' => 'Contact',
+ *                'url' => '/contact',
+ *            ],
+ *            [
+ *                'label' => 'Login',
+ *                'url' => '/login',
+ *            ],
+ *        ];
+ *    }
+ *
+ *    <?php NavBar::begin()
+ *        ->brandLabel('My Application Basic')
+ *        ->brandUrl('/')
+ *        ->options([
+ *            'class' => 'navbar navbar-dark bg-dark navbar-expand-lg text-white',
+ *        ])
+ *        ->init();
```

Comment:
```suggestion
 *        ->start();
```

Reply:
I think also the `start()` method can be removed

Reply:
done.